spec/lucky_web/number_to_currency_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "../../spec_helper"
+
+private class TestPage
+  include LuckyWeb::HTMLPage
+
+  def render
+  end
+end
+
+describe LuckyWeb::NumberToCurrency do
+  describe "number_to_currency" do
```

Comment:
Is spec format (separate test cases) OK or it'd be better to do them [like Rails with everything put under one test case](https://github.com/rails/rails/blob/19fc41992561c901f66064042aa23f4910101d98/activesupport/test/number_helper_test.rb#L64-L79)?

Reply:
I think what you have here is great :) If it gets much bigger then it might be better to combine things, but for now it's 👍